docs/source/quicktour.md

```diff
@@ -15,6 +15,13 @@ docker run --gpus all --shm-size 1g -p 8080:80 -v $volume:/data \
     --model-id $model
 ```
 
+<Tip>
+If you’re looking to serve gated or private models, which provide
+controlled access to sensitive or proprietary content, check out
+[this guide](https://huggingface.co/docs/text-generation-inference/en/basic_tutorials/gated_model_access)
+for detailed instructions.
+</Tip>
```

Comment:
I think you need a new line after `<Tip>` to make sure URLs are rendered properly. See the preview posted by the bot

```suggestion
<Tip>

If you’re looking to serve gated or private models, which provide
controlled access to sensitive or proprietary content, check out
[this guide](https://huggingface.co/docs/text-generation-inference/en/basic_tutorials/gated_model_access)
for detailed instructions.

</Tip>
```